pajbot/apiwrappers/twitch/helix.py

```diff
@@ -602,3 +602,17 @@ def get_channel_emotes(
             expiry=60 * 60,
             force_fetch=force_fetch,
         )
+
+    def send_chat_announcement(self, bot_id: str, body: Dict[str, str], authorization, channel_id: str = None):
+        if not body:
+            log.error("Invalid call to send_chat_announcement, missing query parameter. message must be specified")
+            return False
+
+        response = self.post(
+            "/chat/announcements",
+            {"broadcaster_id": channel_id, "moderator_id": bot_id},
+            authorization=authorization,
+            json=body,
+        )
+
+        return response.status_code == 204
```

Comment:
Don‘t return `True` for success here. Return nothing (`-> None` in the type declaration). `self.post` already raises an exception in case the API request failed with a bad status code.

Reply:
In that case, don`t do `response = `, just call `self.post()` without using the returned value.